__tests__/app.test.js

```diff
@@ -109,6 +109,47 @@ describe("App", () => {
             });
         });
     });
+    describe("GET /api/articles/:article_id", () => {
+        it("Check if endpoint returns an single article object", () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toEqual(expect.objectContaining({}));
+            });
+        });
+        it("Check if returned single article object is correct and matches the corect format", () => {
+            const secondArticle =   {
+                article_id : 2,
```

Comment:
You could expect the exact article_id back like you have done here, but the rest of the properties you could just expect back a string, it would save you time. But it works, so well done 😃